en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md

```diff
@@ -106,28 +112,38 @@ throttle out requests. This will put an unexpected load on the backend. To addre
 supports distributed throttling where it is able to work with a central traffic management solution. In this case,
 multiple Choreo Connect instances can connect with WSO2 API Manager
 ([WSO2 Traffic Manager]({{base_path}}/install-and-setup/setup/distributed-deployment/product-profiles)
-and perform rate-limiting precisely. Find information on how to enable distributed rate limiting from [here](https://mg.docs.wso2.com/en/latest/publish/rate-limiting/distributed-throttling/#distributed-throttling).
+and perform rate-limiting precisely. Find information on how to enable distributed rate limiting from [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/rate-limiting/distributed-throttling).
 
 !!! note
     If you start the WSO2 API Manager without providing any profile, it runs as All in One Node (All the profiles are activated). For testing purposes, you can simply start the API Manager following the [quick start guide]({{base_path}}/getting-started/quick-start-guide) and test.
 
 ### Different levels of throttling
 
+#### Advanced rate limiting (API publisher)
+
+Advanced rate limiting policies are applied when you are Publishing an API. It can be further divided into the following two levels base on the applicability.
```

Comment:
```suggestion
Advanced rate limiting policies are applied when you are Publishing an API. It can be further divided into the following two levels based on the applicability.
```